Cohere.IntegrationTests/CohereStepDefinitions.cs

```diff
@@ -0,0 +1,40 @@
+using Reqnroll;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Cohere.IntegrationTests;
+
+[Binding]
+
+/// <summary>
+/// Defines the step definitions used by all Cohere integration tests
+/// </summary>
+public partial class CohereStepDefinitions
+{
+    protected  CohereClient? _client;
```

Comment:
no need to be protected here. Partial class is different from inheritance. You can just use private